apps/minifront/src/components/staking/account/delegation-value-view/staking-actions/form-dialog.tsx

```diff
@@ -42,7 +42,7 @@ export const FormDialog = ({
    * A `ValueView` representing the address's balance of staking (UM) tokens.
    * Used to show the user how many tokens they have available to delegate.
    */
-  unstakedTokens?: ValueView;
+  stakingTokens?: ValueView;
```

Comment:
Not strictly part of this PR, but we had a mix of the terms "unstaked tokens" and "staking tokens," and both refer to the same thing, so I figured "staking tokens" is clearer